### PR TITLE
Add link to Generating Candid for Rust doc

### PR DESCRIPTION
**Pull request by jennifertrin**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [X] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [X] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

**Comment by jessiemongeon1** [comment]
@jennifertrin The link you've added points to David's blog post about using the IC Rust CDK. The IC Rust CDK (https://github.com/dfinity/cdk-rs) isn't a community developed tool, and we have an existing guide on the Rust Candid generation using that CDK here: https://internetcomputer.org/docs/current/developer-docs/backend/candid/generating-candid

Maybe it would be better to edit this page to link to that guide instead?

File: docs/developer-docs/backend/candid/candid-howto.md [hunk 1]
```diff
@@ -20,7 +20,9 @@ Candid types can be used to describe a service via a Candid service description
 
 If you write a canister in Motoko, for example, the compiler automatically generates a Candid description when you compile the program. If you use the SDK, you will typically see the auto-generated `.did` files in the `/declarations` directory of your project. Since these files are are auto-generated, it is recommended they should not be manually edited. Even if you change the `.did` files in your project, they will be overwritten in the next dfx build.
 
-In other languages, like Rust, you will have to write the Candid interface description manually. With the help of types, we developed tools to automatically generate UI and perform random testing based on the service description file. 
+In other languages, like Rust, you will have to write the Candid interface description manually. However, there is a [community-built script](https://daviddalbusco.com/blog/automatic-candid-generation-in-rust-exploring-the-ic-cdk-v0-10-0-update/) that can automatically generate Candid interface descriptions for Rust.
```
Comment (jessiemongeon1) [inline docs/developer-docs/backend/candid/candid-howto.md:23]:
```suggestion
For canisters written in Rust, the Candid extractor tool can be used with the IC CDK version `0.11.0` and newer. You can find the full instructions for using the Candid extractor [here](/docs/current/developer-docs/backend/candid/generating-candid). 

In other languages, you will have to write the Candid interface description manually. With the help of types, we have developed tools to automatically generate UI and perform random testing based on the service description file. 
```

File: docs/developer-docs/backend/candid/candid-howto.md [hunk 2]
```diff
@@ -20,7 +20,9 @@ Candid types can be used to describe a service via a Candid service description
...
+
+With the help of types, we developed tools to automatically generate UI and perform random testing based on the service description file.
```
Comment (jessiemongeon1) [inline docs/developer-docs/backend/candid/candid-howto.md:25]:
```suggestion

```

**Review by jessiemongeon1 — APPROVED** [review]

**Comment by jennifertrin** [comment]
> @jennifertrin The link you've added points to David's blog post about using the IC Rust CDK. The IC Rust CDK (https://github.com/dfinity/cdk-rs) isn't a community developed tool, and we have an existing guide on the Rust Candid generation using that CDK here: https://internetcomputer.org/docs/current/developer-docs/backend/candid/generating-candid
> 
> Maybe it would be better to edit this page to link to that guide instead?

Apologies for the late reply here! Thank you for the clarification here. I agree with your updates and how you've explained how to CDK to get the candid file.

**Pull request merged**